lib/oc-tests/test_shareDir.py

```diff
@@ -95,6 +95,13 @@ def setup(step):
     check_users(config.oc_number_test_users)
 
     reset_rundir()
+    reset_server_log_file()
```

Comment:
Suppose I merge this branch into master. What is the sequence of events you prefer?

Do you want first re-create your pull request, I merge this and then we do point 2)
Or do you want me to do point 2), merge it to the master and then you re-create your pull request?

2) separate some of your specific functions from init.py so it would be easier to work with pull requests